### ISSUE TITLE
Install fails with looping .crc/machines/crc/id_rsa: no such file or directory

### ISSUE THREAD

**Issue by davidkarlsen** [code-ready/crc#1116]
### General information

  * OS: Mac
  * Hypervisor: hyperkit
  * Did you run `crc setup` before starting it (Yes/No)?: Yes

## CRC version
```bash
crc version
crc version: 1.7.0+fa7e558
OpenShift version: 4.3.1 (embedded in binary)

have also attempted several other 4.x versions
```
  
## CRC status
```bash
crc status
ERRO error: stat /Users/et2448/.crc/machines/crc/kubeconfig: no such file or directory
 - exit status 1 
```

## CRC config
```bash
<no output>
```

## Host Operating System
```bash
sw_vers
ProductName:    Mac OS X
ProductVersion: 10.15.3
BuildVersion:   19D76

```

### Steps to reproduce

  1. download crc binary, clear the security for the binary system prefs -> security
  2. crc setup
  3. crc start will hang as described in "actual"

### Expected
should complete install, or at least timeout and point at some failing step

### Actual

INFO Creating CodeReady Containers VM for OpenShift 4.3.1... 

### Logs

when attempting start for 2nd time with log I get to this step:
```
DEBU error: Temporary Error: stat /Users/et2448/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s 
DEBU retry loop 57                                
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/et2448/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s 
DEBU retry loop 58                                
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/et2448/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s 
DEBU retry loop 59                                
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/et2448/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s 
ERRO Failed to connect to the CRC VM with SSH -- host might be unreachable 
Making call to close driver server
(crc) Calling .Close
(crc) DBG | Closing plugin on server side
Successfully made call to close driver server
Making call to close connection to plugin binary
Making call to close driver server
(crc) Calling .Close
Successfully made call to close driver server
Making call to close connection to plugin binary
(crc) DBG | Closing plugin on server side
```

**Comment by praveenkumar** [comment]
@davidkarlsen Can you do following and also put complete logs to a gist ?
```
$ crc delete
$ rm -fr ~/.crc
$ crc setup
$ crc start --log-level debug
```

**Comment by gbraad** [comment]
> OS: Max

We do not support this operating system... is that a hackintosh clone? ;-)

> DEBU error: Temporary Error: stat /Users/et2448/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s 


On start of the VM we replace the standard key with a user (machine specific one), please try `crc delete` before trying again

**Comment by davidkarlsen** [comment]
lol - yes, it's a typo-clone :-) - fixed
I've attempted install several times and always delete ~/.crc
I'll get back with the updates

**Comment by gbraad** [comment]
we need a little bit more information. especially versions of the OS, full debug log... the listing (permissions) on the files in your `~/.crc` folder.

**Comment by davidkarlsen** [comment]
[log.txt](https://github.com/code-ready/crc/files/4359956/log.txt)

**Comment by werne2j** [comment]
I am seeing a similar issue..

MacOS 1.15.3

crc version
```
crc version: 1.6.0+8ef676f
OpenShift version: 4.3.0 (embedded in binary)
```

Also tried crc version 1.7 with no luck

ls -al ~/.crc/machines/crc
<img width="685" alt="Screen Shot 2020-03-25 at 4 40 38 PM" src="https://user-images.githubusercontent.com/5973248/77583427-6de4c080-6eb7-11ea-9be9-9021490c81b0.png">


```
INFO Extracting bundle: crc_hyperkit_4.3.0.crcbundle ...
DEBU Uncompressing /Users/starship/.crc/cache/crc_hyperkit_4.3.0.crcbundle to /Users/jake/.crc/cache
INFO Creating CodeReady Containers VM for OpenShift 4.3.0...
Found binary path at /Users/starship/.crc/bin/crc-driver-hyperkit
Launching plugin server for driver hyperkit
Plugin server listening at address 127.0.0.1:59975
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
(crc) Calling .GetMachineName
(crc) Calling .DriverName
Running pre-create checks...
(crc) Calling .PreCreateCheck
(crc) DBG | exe=/Users/starship/.crc/bin/crc-driver-hyperkit uid=0
(crc) Calling .GetConfigRaw
Creating machine...
(crc) Calling .Create
(crc) DBG | exe=/Users/starship/.crc/bin/crc-driver-hyperkit uid=0
(crc) Downloading /Users/starship/.crc/cache/crc.disk from file:///Users/starship/.crc/cache/crc_hyperkit_4.3.0/crc.qcow2...
(crc) DBG | exe=/Users/starship/.crc/bin/crc-driver-hyperkit uid=0
(crc) DBG | clean start, hyperkit pid file doesn't exist: /Users/starship/.crc/machines/crc/hyperkit.pid
(crc) DBG | Using hyperkit binary from /Users/starship/.crc/bin/hyperkit
(crc) DBG | Using UUID c3d68012-0208-11ea-9fd7-f2189899ab08
(crc) DBG | Generated MAC e:f0:e3:c7:4f:be
(crc) DBG | Starting with cmdline: BOOT_IMAGE=(hd0,gpt1)/ostree/rhcos-9a8c2fd60f5219ecf823c685ed70097b8c55bb8b0d43e4192737ea7918630918/vmlinuz-4.18.0-147.3.1.el8_1.x86_64 rhcos.root=crypt_rootfs console=tty0 console=ttyS0,115200n8 ignition.platform.id=qemu rd.luks.options=discard ostree=/ostree/boot.0/rhcos/9a8c2fd60f5219ecf823c685ed70097b8c55bb8b0d43e4192737ea7918630918/0
(crc) DBG | Trying to execute /Users/starship/.crc/bin/hyperkit -A -u -F /Users/starship/.crc/machines/crc/hyperkit.pid -c 4 -m 8192M -s 0:0,hostbridge -s 31,lpc -s 1:0,virtio-net -U c3d68012-0208-11ea-9fd7-f2189899ab08 -s 2:0,virtio-blk,file:///Users/starship/.crc/machines/crc/crc.disk,format=qcow -s 3,virtio-rnd -l com1,autopty=/Users/starship/.crc/machines/crc/tty,log=/Users/starship/.crc/machines/crc/console-ring -f kexec,/Users/starship/.crc/cache/crc_hyperkit_4.3.0/vmlinuz-4.18.0-147.3.1.el8_1.x86_64,/Users/starship/.crc/cache/crc_hyperkit_4.3.0/initramfs-4.18.0-147.3.1.el8_1.x86_64.img,earlyprintk=serial BOOT_IMAGE=(hd0,gpt1)/ostree/rhcos-9a8c2fd60f5219ecf823c685ed70097b8c55bb8b0d43e4192737ea7918630918/vmlinuz-4.18.0-147.3.1.el8_1.x86_64 rhcos.root=crypt_rootfs console=tty0 console=ttyS0,115200n8 ignition.platform.id=qemu rd.luks.options=discard ostree=/ostree/boot.0/rhcos/9a8c2fd60f5219ecf823c685ed70097b8c55bb8b0d43e4192737ea7918630918/0
(crc) DBG | exe=/Users/starship/.crc/bin/crc-driver-hyperkit uid=0
(crc) DBG | hyperkit pid from json: 50418
(crc) DBG | Searching for e:f0:e3:c7:4f:be in /var/db/dhcpd_leases ...
(crc) DBG | Found 1 entries in /var/db/dhcpd_leases!
(crc) DBG | dhcp entry: {Name:crc-w6th5-master-0 IPAddress:192.168.64.2 HWAddress:e:f0:e3:c7:4f:be ID:1,e:f0:e3:c7:4f:be Lease:0x5e7d0fb3}
(crc) DBG | Found match: e:f0:e3:c7:4f:be
(crc) DBG | IP: 192.168.64.2
(crc) Calling .GetConfigRaw
(crc) Calling .DriverName
(crc) Calling .DriverName
Waiting for machine to be running, this may take a few minutes...
(crc) Calling .GetState
(crc) DBG | exe=/Users/starship/.crc/bin/crc-driver-hyperkit uid=0
(crc) DBG | hyperkit pid from json: 50418
Machine is up and running!
Machine successfully created
Created /Users/starship/.crc/machines/crc/.crc-exist
(crc) Calling .GetState
(crc) DBG | exe=/Users/starship/.crc/bin/crc-driver-hyperkit uid=0
(crc) DBG | hyperkit pid from json: 50418
Found binary path at /Users/starship/.crc/bin/crc-driver-hyperkit
Launching plugin server for driver hyperkit
Plugin server listening at address 127.0.0.1:59981
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
DEBU Waiting until ssh is available
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
Using SSH private key: /Users/starship/.crc/cache/crc_hyperkit_4.3.0/id_rsa_crc (-r--------)
&{[-F /dev/null -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=no -o ControlPath=none -o LogLevel=quiet -o PasswordAuthentication=no -o ServerAliveInterval=60 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null core@192.168.64.2 -o IdentitiesOnly=yes -i /Users/starship/.crc/cache/crc_hyperkit_4.3.0/id_rsa_crc -p 22] /usr/bin/ssh <nil>}
DEBU About to run SSH command:
exit 0
```

**Comment by werne2j** [comment]
Looks like i was able to get start to finish by disconnecting from my VPN

**Comment by gbraad** [comment]
Invalid. Known issue - VPN

**Comment by owen-sun** [comment]
@davidkarlsen how did you fix the issue? I met same issue in my macos too.

owen@localhost crc_hyperkit_4.3.10 % crc start --log-level debug
DEBU No new version available. The latest version is 1.9.0
INFO Checking if oc binary is cached
DEBU oc binary already cached
INFO Checking if podman remote binary is cached
DEBU podman remote binary already cached
INFO Checking if running as non-root
INFO Checking if HyperKit is installed
DEBU Checking if hyperkit is installed
INFO Checking if crc-driver-hyperkit is installed
DEBU Checking if crc-driver-hyperkit is installed
DEBU Running '/Users/owen/.crc/bin/crc-driver-hyperkit version'
DEBU crc-driver-hyperkit is already installed in /Users/owen/.crc/bin/crc-driver-hyperkit
INFO Checking file permissions for /etc/resolver/testing
INFO Checking file permissions for /etc/hosts
Checking file: /Users/owen/.crc/machines/crc/.crc-exist
Found binary path at /Users/owen/.crc/bin/crc-driver-hyperkit
Launching plugin server for driver hyperkit
Plugin server listening at address 127.0.0.1:50187
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
(crc) Calling .GetBundleName
(crc) Calling .GetState
(crc) DBG | exe=/Users/owen/.crc/bin/crc-driver-hyperkit uid=0
(crc) DBG | hyperkit pid from json: 29557
(crc) DBG | hyperkit pid 29557 missing from process table
INFO Starting CodeReady Containers VM for OpenShift 4.3.10...
(crc) Calling .Start
(crc) DBG | exe=/Users/owen/.crc/bin/crc-driver-hyperkit uid=0
(crc) DBG | clean start, hyperkit pid file doesn't exist: /Users/owen/.crc/machines/crc/hyperkit.pid
(crc) DBG | Using hyperkit binary from /Users/owen/.crc/bin/hyperkit
(crc) DBG | Using UUID c3d68012-0208-11ea-9fd7-f2189899ab08
(crc) DBG | Generated MAC e:f0:e3:c7:4f:be
(crc) DBG | Starting with cmdline: BOOT_IMAGE=(hd0,gpt1)/ostree/rhcos-27bcceba8a2fcb7ffbb20f157cdc45d881b19fcbca0e4a79d5d6b9b7d3428571/vmlinuz-4.18.0-147.5.1.el8_1.x86_64 rhcos.root=crypt_rootfs console=tty0 console=ttyS0,115200n8 ignition.platform.id=qemu rd.luks.options=discard ostree=/ostree/boot.1/rhcos/27bcceba8a2fcb7ffbb20f157cdc45d881b19fcbca0e4a79d5d6b9b7d3428571/0
(crc) DBG | Trying to execute /Users/owen/.crc/bin/hyperkit -A -u -F /Users/owen/.crc/machines/crc/hyperkit.pid -c 4 -m 8192M -s 0:0,hostbridge -s 31,lpc -s 1:0,virtio-net -U c3d68012-0208-11ea-9fd7-f2189899ab08 -s 2:0,virtio-blk,file:///Users/owen/.crc/machines/crc/crc.disk,format=qcow -s 3,virtio-rnd -l com1,autopty=/Users/owen/.crc/machines/crc/tty,log=/Users/owen/.crc/machines/crc/console-ring -f kexec,/Users/owen/.crc/cache/crc_hyperkit_4.3.10/vmlinuz-4.18.0-147.5.1.el8_1.x86_64,/Users/owen/.crc/cache/crc_hyperkit_4.3.10/initramfs-4.18.0-147.5.1.el8_1.x86_64.img,earlyprintk=serial BOOT_IMAGE=(hd0,gpt1)/ostree/rhcos-27bcceba8a2fcb7ffbb20f157cdc45d881b19fcbca0e4a79d5d6b9b7d3428571/vmlinuz-4.18.0-147.5.1.el8_1.x86_64 rhcos.root=crypt_rootfs console=tty0 console=ttyS0,115200n8 ignition.platform.id=qemu rd.luks.options=discard ostree=/ostree/boot.1/rhcos/27bcceba8a2fcb7ffbb20f157cdc45d881b19fcbca0e4a79d5d6b9b7d3428571/0
(crc) DBG | exe=/Users/owen/.crc/bin/crc-driver-hyperkit uid=0
(crc) DBG | hyperkit pid from json: 29602
(crc) DBG | Searching for e:f0:e3:c7:4f:be in /var/db/dhcpd_leases ...
(crc) DBG | Found 1 entries in /var/db/dhcpd_leases!
(crc) DBG | dhcp entry: {Name:crc-bq5fv-master-0 IPAddress:192.168.64.2 HWAddress:e:f0:e3:c7:4f:be ID:1,e:f0:e3:c7:4f:be Lease:0x5ea8daf4}
(crc) DBG | Found match: e:f0:e3:c7:4f:be
(crc) DBG | IP: 192.168.64.2
(crc) Calling .GetConfigRaw
(crc) Calling .GetState
(crc) DBG | exe=/Users/owen/.crc/bin/crc-driver-hyperkit uid=0
(crc) DBG | hyperkit pid from json: 29602
Found binary path at /Users/owen/.crc/bin/crc-driver-hyperkit
Launching plugin server for driver hyperkit
Plugin server listening at address 127.0.0.1:50191
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
DEBU Waiting until ssh is available
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 1
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 2
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 3
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 4
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 5
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 6
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 7
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 8
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 9
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 10
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 11
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 12
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 13
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 14
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 15
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 16
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 17
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 18
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 19
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 20
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 21
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 22
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 23
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 24
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 25
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 26
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 27
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 28
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 29
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 30
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 31
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 32
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 33
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 34
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 35
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 36
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 37
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 38
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 39
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 40
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 41
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 42
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 43
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 44
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 45
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 46
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 47
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 48
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 49
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 50
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 51
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 52
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 53
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 54
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 55
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 56
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 57
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 58
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
DEBU retry loop 59
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
<nil>
DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - sleeping 1s
ERRO Failed to connect to the CRC VM with SSH -- host might be unreachable
Making call to close driver server
(crc) Calling .Close
(crc) DBG | Closing plugin on server side
Successfully made call to close driver server
Making call to close connection to plugin binary
Making call to close driver server
(crc) Calling .Close
Successfully made call to close driver server
Making call to close connection to plugin binary
(crc) DBG | Closing plugin on server side
owen@localhost crc_hyperkit_4.3.10 %

**Comment by davidkarlsen** [comment]
@owen-sun it was never fixed. I gave up. Guess they're not code ready after all...

**Comment by gbraad** [comment]
That does not really help with debugging the issue.

@owen-sun 
The error `1DEBU error: Temporary Error: stat /Users/owen/.crc/machines/crc/id_rsa: no such file or directory - ` indicates something failed during the initial generation of the machien specific key. Can you do a

```
crc delete -f
crc cleanup
crc setup
crc start
```

**Comment by gbraad** [comment]
@praveenkumar Do you have any idea what could cause this? I am at the moment not on the mac

**Comment by praveenkumar** [comment]
> Do you have any idea what could cause this? I am at the moment not on the mac

looking at the logs and issue which @davidkarlsen faced might be because of host doesn't allow to ssh to the VM (may be vpn) and that is the reason initial ssh command failed.

@owen-sun can you try to follow what https://github.com/code-ready/crc/issues/1116#issuecomment-620388863 suggested and put the debug logs in a gist for analysis.

**Comment by owen-sun** [comment]
@gbraad @praveenkumar I tried above steps and it still doesn't work, here are new errors:
owen@localhost ~ % crc start --log-level debug
DEBU No new version available. The latest version is 1.9.0
INFO Checking if oc binary is cached
DEBU oc binary already cached
INFO Checking if podman remote binary is cached
DEBU podman remote binary already cached
INFO Checking if running as non-root
INFO Checking if HyperKit is installed
DEBU Checking if hyperkit is installed
INFO Checking if crc-driver-hyperkit is installed
DEBU Checking if crc-driver-hyperkit is installed
DEBU Running '/Users/owen/.crc/bin/crc-driver-hyperkit version'
DEBU crc-driver-hyperkit is already installed in /Users/owen/.crc/bin/crc-driver-hyperkit
INFO Checking file permissions for /etc/resolver/testing
INFO Checking file permissions for /etc/hosts
Checking file: /Users/owen/.crc/machines/crc/.crc-exist
? Image pull secret [? for help] *********************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************INFO Loading bundle: crc_hyperkit_4.3.10.crcbundle ... *******************************************************************************************************INFO Checking size of the disk image /Users/owen/.crc/cache/crc_hyperkit_4.3.10/crc.qcow2 ... ****************************************************************INFO Creating CodeReady Containers VM for OpenShift 4.3.10... ************************************************************************************************Found binary path at /Users/owen/.crc/bin/crc-driver-hyperkit*************************************************************************************************Launching plugin server for driver hyperkit*******************************************************************************************************************Plugin server listening at address 127.0.0.1:53018************************************************************************************************************() Calling .GetVersion****************************************************************************************************************************************Using API Version  1******************************************************************************************************************************************() Calling .SetConfigRaw**************************************************************************************************************************************() Calling .GetMachineName************************************************************************************************************************************(crc) Calling .GetMachineName*********************************************************************************************************************************(crc) Calling .DriverName*************************************************************************************************************************************Running pre-create checks...**********************************************************************************************************************************(crc) Calling .PreCreateCheck*********************************************************************************************************************************(crc) DBG | exe=/Users/owen/.crc/bin/crc-driver-hyperkit uid=0***************
(crc) Calling .GetConfigRaw
Creating machine...
(crc) Calling .Create
(crc) DBG | exe=/Users/owen/.crc/bin/crc-driver-hyperkit uid=0
(crc) Downloading /Users/owen/.crc/cache/crc.disk from file:///Users/owen/.crc/cache/crc_hyperkit_4.3.10/crc.qcow2...
(crc) DBG | exe=/Users/owen/.crc/bin/crc-driver-hyperkit uid=0
(crc) DBG | clean start, hyperkit pid file doesn't exist: /Users/owen/.crc/machines/crc/hyperkit.pid
(crc) DBG | Using hyperkit binary from /Users/owen/.crc/bin/hyperkit
(crc) DBG | Using UUID c3d68012-0208-11ea-9fd7-f2189899ab08
(crc) DBG | Generated MAC e:f0:e3:c7:4f:be
(crc) DBG | Starting with cmdline: BOOT_IMAGE=(hd0,gpt1)/ostree/rhcos-27bcceba8a2fcb7ffbb20f157cdc45d881b19fcbca0e4a79d5d6b9b7d3428571/vmlinuz-4.18.0-147.5.1.el8_1.x86_64 rhcos.root=crypt_rootfs console=tty0 console=ttyS0,115200n8 ignition.platform.id=qemu rd.luks.options=discard ostree=/ostree/boot.1/rhcos/27bcceba8a2fcb7ffbb20f157cdc45d881b19fcbca0e4a79d5d6b9b7d3428571/0
(crc) DBG | Trying to execute /Users/owen/.crc/bin/hyperkit -A -u -F /Users/owen/.crc/machines/crc/hyperkit.pid -c 4 -m 8192M -s 0:0,hostbridge -s 31,lpc -s 1:0,virtio-net -U c3d68012-0208-11ea-9fd7-f2189899ab08 -s 2:0,virtio-blk,file:///Users/owen/.crc/machines/crc/crc.disk,format=qcow -s 3,virtio-rnd -l com1,autopty=/Users/owen/.crc/machines/crc/tty,log=/Users/owen/.crc/machines/crc/console-ring -f kexec,/Users/owen/.crc/cache/crc_hyperkit_4.3.10/vmlinuz-4.18.0-147.5.1.el8_1.x86_64,/Users/owen/.crc/cache/crc_hyperkit_4.3.10/initramfs-4.18.0-147.5.1.el8_1.x86_64.img,earlyprintk=serial BOOT_IMAGE=(hd0,gpt1)/ostree/rhcos-27bcceba8a2fcb7ffbb20f157cdc45d881b19fcbca0e4a79d5d6b9b7d3428571/vmlinuz-4.18.0-147.5.1.el8_1.x86_64 rhcos.root=crypt_rootfs console=tty0 console=ttyS0,115200n8 ignition.platform.id=qemu rd.luks.options=discard ostree=/ostree/boot.1/rhcos/27bcceba8a2fcb7ffbb20f157cdc45d881b19fcbca0e4a79d5d6b9b7d3428571/0
(crc) DBG | exe=/Users/owen/.crc/bin/crc-driver-hyperkit uid=0
(crc) DBG | hyperkit pid from json: 4029
(crc) DBG | Searching for e:f0:e3:c7:4f:be in /var/db/dhcpd_leases ...
(crc) DBG | Found 1 entries in /var/db/dhcpd_leases!
(crc) DBG | dhcp entry: {Name:crc-bq5fv-master-0 IPAddress:192.168.64.2 HWAddress:e:f0:e3:c7:4f:be ID:1,e:f0:e3:c7:4f:be Lease:0x5ea927fa}
(crc) DBG | Found match: e:f0:e3:c7:4f:be
(crc) DBG | IP: 192.168.64.2
(crc) Calling .GetConfigRaw
(crc) Calling .DriverName
(crc) Calling .DriverName
Waiting for machine to be running, this may take a few minutes...
(crc) Calling .GetState
(crc) DBG | exe=/Users/owen/.crc/bin/crc-driver-hyperkit uid=0
(crc) DBG | hyperkit pid from json: 4029
Machine is up and running!
Machine successfully created
Created /Users/owen/.crc/machines/crc/.crc-exist
(crc) Calling .GetState
(crc) DBG | exe=/Users/owen/.crc/bin/crc-driver-hyperkit uid=0
(crc) DBG | hyperkit pid from json: 4029
Found binary path at /Users/owen/.crc/bin/crc-driver-hyperkit
Launching plugin server for driver hyperkit
Plugin server listening at address 127.0.0.1:53024
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
DEBU Waiting until ssh is available
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
Using SSH private key: /Users/owen/.crc/cache/crc_hyperkit_4.3.10/id_rsa_crc (-rw-------)
&{[-F /dev/null -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=no -o ControlPath=none -o LogLevel=quiet -o PasswordAuthentication=no -o ServerAliveInterval=60 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null core@192.168.64.2 -o IdentitiesOnly=yes -i /Users/owen/.crc/cache/crc_hyperkit_4.3.10/id_rsa_crc -p 22] /usr/bin/ssh <nil>}
DEBU About to run SSH command:
exit 0
DEBU SSH cmd err, output: exit status 255:
DEBU error: Temporary Error: ssh command error:
command : exit 0
err     : exit status 255
output  :  - sleeping 1s
DEBU retry loop 1
(crc) Calling .GetSSHHostname
(crc) Calling .GetSSHPort
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHKeyPath
(crc) Calling .GetSSHUsername
Using SSH client type: external
Using SSH private key: /Users/owen/.crc/cache/crc_hyperkit_4.3.10/id_rsa_crc (-rw-------)
&{[-F /dev/null -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=no -o ControlPath=none -o LogLevel=quiet -o PasswordAuthentication=no -o ServerAliveInterval=60 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null core@192.168.64.2 -o IdentitiesOnly=yes -i /Users/owen/.crc/cache/crc_hyperkit_4.3.10/id_rsa_crc -p 22] /usr/bin/ssh <nil>}
DEBU About to run SSH command:
exit 0

**Comment by cfergeau** [comment]
Please paste the full log. @davidkarlsen can you also follow Praveen's instructions and paste a full log of your interactions with crc?

**Comment by gbraad** [comment]
https://github.com/code-ready/crc/issues/1116#issuecomment-616974107 

Reopened

**Comment by owen-sun** [comment]
@cfergeau @gbraad here is the full log, it still doesn't work after I tried delete, setup, start over 10 times, appreciate your guidance.

[crc.log](https://github.com/code-ready/crc/files/4544948/crc.log)

**Comment by gbraad** [comment]
> might be because of host doesn't allow to ssh to the VM (may be vpn) 

We might want to add something to the issue template to get this info for macOS. We do collect this fro Windows for instance.

**Comment by cfergeau** [comment]
```
ERRO Failed to query DNS from host: lookup api.crc.testing on 192.168.1.1:53: read udp 192.168.1.101:62562->192.168.1.1:53: i/o timeout
```
This is the part which I don't really understand. It should be looking up `api.crc.testing` on `192.168.64.2` which is the VM IP. Do you have a firewall/antivirus/.... running which could interfere with the network?

**Comment by owen-sun** [comment]
@cfergeau I have VPN connected, no special setting on firewall/antivirus.

**Comment by cfergeau** [comment]
Can you try again after disconnecting from the VPN?

**Comment by davidkarlsen** [comment]
@cfergeau It was added in https://github.com/code-ready/crc/issues/1116#issuecomment-601719444

**Comment by gbraad** [comment]
> I have VPN connected, no special setting on firewall/antivirus.

As recorded in the documentation and as a known issue, VPN connectivity can interfere with the start and communication with the cluster. Also see comment: https://github.com/code-ready/crc/issues/1116#issuecomment-604619309 on this issue.

**Comment by gbraad** [comment]
> @cfergeau It was added in [#1116 (comment)](https://github.com/code-ready/crc/issues/1116#issuecomment-601719444)

```
SSH cmd err, output: exit status 255:        
error: Temporary Error: ssh command error:
command : exit 0
err     : exit status 255
output  :  - sleeping 1s 
etry loop 6                                 
About to run SSH command:
exit 0
```

For this it would be best to try `ssh -i ~/.crc/machines/crc/id_rsa core@[vmip]` and check why the ssh command fails. It exits and loops....

**Comment by cfergeau** [comment]
> @cfergeau It was added in [#1116 (comment)](https://github.com/code-ready/crc/issues/1116#issuecomment-601719444)

This seems partial as it ends at
```
[37mDEBU[0m SSH cmd err, output: exit status 255:        
[37mDEBU[0m error: Temporary Error: ssh command error:
command : exit 0
err     : exit status 255
output  :  - sleeping 1s 
[37mDEBU[0m retry loop 7                                 
[37mDEBU[0m About to run SSH command:
exit 0             
```
it usually retries 30, 60 or 90 times before timing out, here it says `retry loop 7`

**Comment by kuwv** [comment]
Experienced this on Fedora 30 after repeatedly created and deleted the instance. Turned out to be a caching issue with qemu which required a restart with libvirtd.

This may be a similar issue with xhyve.

**Comment by AndrienkoAleksandr** [comment]
I see the same. Crc version:
```
$ crc version
CodeReady Containers version: 1.15.0+e317bed
OpenShift version: 4.5.7 (embedded in binary)

```

**Comment by AndrienkoAleksandr** [comment]
Log:

```
crc-start 
INFO Checking if oc binary is cached              
INFO Checking if podman remote binary is cached   
INFO Checking if goodhosts binary is cached       
INFO Checking minimum RAM requirements            
INFO Checking if running as non-root              
INFO Checking if Virtualization is enabled        
INFO Checking if KVM is enabled                   
INFO Checking if libvirt is installed             
INFO Checking if user is part of libvirt group    
INFO Checking if libvirt daemon is running        
INFO Checking if a supported libvirt version is installed 
INFO Checking if crc-driver-libvirt is installed  
INFO Checking if libvirt 'crc' network is available 
INFO Checking if libvirt 'crc' network is active  
INFO Checking if NetworkManager is installed      
INFO Checking if NetworkManager service is running 
INFO Checking if /etc/NetworkManager/conf.d/crc-nm-dnsmasq.conf exists 
INFO Checking if /etc/NetworkManager/dnsmasq.d/crc.conf exists 
INFO Starting CodeReady Containers VM for OpenShift 4.5.7... 
ERRO Failed to connect to the CRC VM with SSH -- host might be unreachable: Temporary Error: stat /home/user/.crc/machines/crc/id_rsa: no such file or directory
Temporary Error: stat /home/user/.crc/machines/crc/id_rsa: no such file or directory
```

**Comment by AndrienkoAleksandr** [comment]
And really there is no file /home/user/.crc/machines/crc/id_rsa

**Comment by AndrienkoAleksandr** [comment]
OS: Fedora 32 !
Hypervisor: hyperkit

**Comment by AndrienkoAleksandr** [comment]
Reinstall crc each time to make it working after restart laptop, looks weird...

**Comment by cfergeau** [comment]
Can you open a new issue for this? It's probably a different root cause

**Comment by AndrienkoAleksandr** [comment]
Ok, I will
